code/game/turfs/turf.dm

```diff
@@ -303,3 +322,9 @@ var/const/enterloopsanity = 100
 		if(isliving(AM))
 			var/mob/living/M = AM
 			M.turf_collision(src, speed)
+
+/turf/proc/is_open()
```

Comment:
I already added a var, is_hole, for this

unless complex functionality is needed, checking that var is probably faster. could at least return it here

Reply:
> I already added a var, is_hole, for this
> 
> unless complex functionality is needed, checking that var is probably faster. could at least return it here

Deleted